### PR TITLE
291 climate

### LINKED ISSUE #291: config code climete
- config test coverage
- config code maintainability
- put it in the readme

### PR DESCRIPTION
**Pull request by IuryNogueira**
fix #291

### CODE REVIEW

File: .semaphore/semaphore.yml [hunk 1]
```diff
@@ -19,51 +19,54 @@ blocks:
...
             - cache restore
-            - yarn lint
-
-  - name: Build
+            - yarn install
+            - curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
+            - chmod +x ./cc-test-reporter
+            - ./cc-test-reporter before-build
+            - yarn test:coverage
+            - ls
```
Comment (edimossilva) [inline .semaphore/semaphore.yml]: remove this ls

File: .semaphore/semaphore.yml [hunk 2]
```diff
@@ -19,51 +19,54 @@ blocks:
...
+            - ./cc-test-reporter format-coverage -t lcov -o codeclimate.frontend.json coverage/lcov.info
+            - ./cc-test-reporter sum-coverage codeclimate.*.json -o codeclimate.total.json
+            -  ./cc-test-reporter upload-coverage -r 901775431df63d8131620226e329cdb757bede481ec12d99b249da870beacb52 -i codeclimate.total.json
```
Comment (edimossilva) [inline .semaphore/semaphore.yml:40]: remove extra space

File: .semaphore/semaphore.yml [hunk 3]
```diff
@@ -19,51 +19,53 @@ blocks:
...
-        - name: lint client
+        - name: test client
           commands:
             - checkout
             - cd frontend
             - cache restore
-            - yarn lint
-
-  - name: Build
+            - yarn install
```
Comment (edimossilva) [inline .semaphore/semaphore.yml:33]: should remove it?

**Review by edimossilva — CHANGES_REQUESTED** [review]

File: .semaphore/semaphore.yml (hunk 3 above)
Comment (IuryNogueira) [reply .semaphore/semaphore.yml:33]: @andersonmalheiro add this line
```
dependencies: ["Install dependencies"]
```
then we can remove the install command. 👍

**Review by edimossilva — APPROVED** [review]
Niceeee
squash and merge

**Pull request merged**